Step by step MailHog Integration

karissekjw opened the pull request:
Hey! Came across your Hanami cookbook where you recommended MailHog. Have managed to integrate it successfully and would love to contribute to your cookbook by updating this section so others can benefit from it too :). MailHog also has a UI view where you can view hanami emails in the browser. Was wondering if you would like that to be included in the Mail preview section as well?

Let me know what else I can do to improve on the proposed changes!

Cheers

karissekjw committed:
Step by step MailHog Integration

Hey! Came across your Hanami cookbook where you recommended MailHog. Have managed to integrate it successfully and would love to contribute to your cookbook by updating this section so others can contribute from it too :). MailHog also has a UI view where you can view hanami emails in the browser. Was wondering if you would like that to be included in the Mail preview section as well?

Let me know what else I can do to improve on the proposed changes!

Cheers

iJackUA commented:
Thanks for contribution!
I added couple comments.

As for email previews as I remember I thought just to cross-post this guide
https://www.davydovanton.com/2016/05/21/preview-hanami-emails-in-browser/
so leave it out of scope for this PR

karissekjw committed:
Update mailhog intergration

- update installation section to refer to mailhog repo
- update hanami mailer config to keep instructions minimal

karissekjw commented:
Thanks for the comments! Definitely learnt something there. I've updated the docs accordingly

iJackUA commented:
Merged, thanks again!
